pages/index.html

```diff
@@ -1,12 +1,27 @@
 <!DOCTYPE html>
```

Comment:
It is a convention to include the index.html in the project root directory level, and additional pages in a directory such as `pages`